docker-image-src/5/neo4jlabs-plugins.json

```diff
@@ -1,7 +1,6 @@
 {
   "apoc": {
     "location": "/var/lib/neo4j/labs/apoc-*-core.jar",
-    "versions": "https://github.com/neo4j/apoc/blob/master/versions.json",
```

Comment:
This shouldn't really be needed, because the file is being used by Neo4j 5.x images which all bundle APOC. I think it's a little confusing for future maintainers to add this "versions" key which isn't gonna be used.